qa/acceptance-tests/src/test/java/io/camunda/it/auth/ClusterVariableAuthorizationIT.java

```diff
@@ -0,0 +1,548 @@
+class ClusterVariableAuthorizationIT {
+
+  @MultiDbTestApplication
+  static final TestStandaloneBroker BROKER =
+      new TestStandaloneBroker().withBasicAuth().withAuthorizationsEnabled();
+
+  private static final String ADMIN = "admin";
+  private static final String CREATE_USER = "createUser";
+  private static final String READ_USER = "readUser";
+  private static final String DELETE_USER = "deleteUser";
+  private static final String NO_PERMISSION_USER = "noPermissionUser";
+  private static final String TEST_TENANT_ID = "test-tenant";
+
+  @UserDefinition
+  private static final TestUser ADMIN_USER =
+      new TestUser(
+          ADMIN,
+          "password",
+          List.of(
+              new Permissions(CLUSTER_VARIABLE, CREATE, List.of("*")),
+              new Permissions(CLUSTER_VARIABLE, READ, List.of("*")),
+              new Permissions(CLUSTER_VARIABLE, DELETE, List.of("*"))));
+
+  @UserDefinition
+  private static final TestUser CREATE_USER_DEF =
+      new TestUser(
+          CREATE_USER,
+          "password",
+          List.of(new Permissions(CLUSTER_VARIABLE, CREATE, List.of("*"))));
+
+  @UserDefinition
+  private static final TestUser READ_USER_DEF =
+      new TestUser(
+          READ_USER, "password", List.of(new Permissions(CLUSTER_VARIABLE, READ, List.of("*"))));
+
+  @UserDefinition
+  private static final TestUser DELETE_USER_DEF =
+      new TestUser(
+          DELETE_USER,
+          "password",
+          List.of(new Permissions(CLUSTER_VARIABLE, DELETE, List.of("*"))));
+
+  @UserDefinition
+  private static final TestUser NO_PERMISSION_USER_DEF =
+      new TestUser(NO_PERMISSION_USER, "password", List.of());
+
+  @BeforeAll
+  static void setUp(@Authenticated(ADMIN) final CamundaClient adminClient) {
+    // Create globally scoped cluster variables
+    adminClient
+        .newGloballyScopedClusterVariableCreateRequest()
+        .create("testVar1", "testValue1")
+        .send()
+        .join();
+
+    adminClient
+        .newGloballyScopedClusterVariableCreateRequest()
+        .create("testVar2", "testValue2")
+        .send()
+        .join();
+
+    adminClient
+        .newGloballyScopedClusterVariableCreateRequest()
+        .create("testVar3", "testValue3")
+        .send()
+        .join();
+
+    // Create tenant-scoped cluster variables
+    adminClient
+        .newTenantScopedClusterVariableCreateRequest(TEST_TENANT_ID)
+        .create("tenantVar1", "tenantValue1")
+        .send()
+        .join();
+
+    adminClient
+        .newTenantScopedClusterVariableCreateRequest(TEST_TENANT_ID)
+        .create("tenantVar2", "tenantValue2")
+        .send()
+        .join();
+
+    adminClient
+        .newTenantScopedClusterVariableCreateRequest(TEST_TENANT_ID)
+        .create("tenantVar3", "tenantValue3")
+        .send()
+        .join();
+
+    waitForClusterVariablesToBeAvailable(adminClient, 6);
+  }
+
+  // ============= READ PERMISSION TESTS =============
```

Comment:
💭 I guess you could introduce a nested test class here to capture this instead of a code comment. Not sure if we have used this with the multi-DB test anywhere yet.